test/runtests.jl

```diff
@@ -165,13 +168,16 @@ end
         ci = first(callsites).info
         @test isa(ci, Cthulhu.UncachedCallInfo)
         @test Cthulhu.is_callsite(ci, ci.wrapped.mi)
-
+        io = IOBuffer()
+        show(io, first(callsites))
+        @test occursin("< uncached >", String(take!(io)))
         # TODO do some test with `LimitedCallInfo`, but they happen at deeper callsites
     end
 end
 
 @testset "union-split constant-prop'ed callsites" begin
     # constant prop' on all the splits
+    io = IOBuffer()
```

Comment:
I'd like to make sure to use different `io` for each test case, in order to eliminate the possibilities of implicit test dependencies (although you're doing `take!(io)` in each test case).

Reply:
Done